benchmarks/rf_use_case/run_benchmark.R

```diff
@@ -0,0 +1,105 @@
+library(mlr3)
+library(data.table)
+library(mlr3torch)
+library(paradox)
+
+library(here)
+
+# define the tasks
```

Comment:
`tsk("oml", task_id = 1067)`
similarly hard-code for every task

this will ignore the OpenML resampling